### ISSUE TITLE
Content type unsupported

### ISSUE THREAD

**Issue by cwaazywabbit** [tagomaru/burp-extension-postman-integration#3]
The following exception is thrown when attempting to export a certain SOAP request.

> 
> java.lang.Exception: [/authws/blah] This content type is not supported.
> 	at assess.ExportProcessor.generateRequest(ExportProcessor.java:154)
> 	at assess.ExportProcessor.process(ExportProcessor.java:64)
> 	at assess.PostmanFrame$1.actionPerformed(PostmanFrame.java:159)
> 	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
> 	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
> 	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
> 	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
> 	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
> 	at java.awt.Component.processMouseEvent(Component.java:6533)
> 	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
> 	at java.awt.Component.processEvent(Component.java:6298)
> 	at java.awt.Container.processEvent(Container.java:2236)
> 	at java.awt.Component.dispatchEventImpl(Component.java:4889)
> 	at java.awt.Container.dispatchEventImpl(Container.java:2294)
> 	at java.awt.Component.dispatchEvent(Component.java:4711)
> 	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
> 	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
> 	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
> 	at java.awt.Container.dispatchEventImpl(Container.java:2280)
> 	at java.awt.Window.dispatchEventImpl(Window.java:2746)
> 	at java.awt.Component.dispatchEvent(Component.java:4711)
> 	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
> 	at java.awt.EventQueue.access$500(EventQueue.java:97)
> 	at java.awt.EventQueue$3.run(EventQueue.java:709)
> 	at java.awt.EventQueue$3.run(EventQueue.java:703)
> 	at java.security.AccessController.doPrivileged(Native Method)
> 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
> 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
> 	at java.awt.EventQueue$4.run(EventQueue.java:731)
> 	at java.awt.EventQueue$4.run(EventQueue.java:729)
> 	at java.security.AccessController.doPrivileged(Native Method)
> 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
> 	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
> 	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
> 	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
> 	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
> 	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
> 	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
> 	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

The request *looks* something like this (I can't give the exact one since it's a client project):

```
POST /authws/blah HTTP/1.1
User-Agent: Mozilla/5.0 (Windows NT 10.0; WOW64; rv:66.0) Gecko/20100101 Firefox/66.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Connection: close
Upgrade-Insecure-Requests: 1
Content-Type: text/xml;charset=UTF-8
Host: domain.com:8443
Content-Length: 318
Authorization: Basic <base64_creds>

<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:oeb="http://something.here/blah">
   <soapenv:Header/>
   <soapenv:Body>
      <oeb:isServiceAlive>
         <!--type: string-->
         <arg></tuv></arg>
      </oeb:isServiceAlive>
   </soapenv:Body>
</soapenv:Envelope>
```

**Comment by tagomaru** [comment]
@cwaazywabbit 
Thx for your raising this up.
I will look into this.

**Comment by tagomaru** [comment]
Hi @cwaazywabbit 
I resolved this with #4 .

Could you try it ?

https://github.com/tagomaru/burp-extension-postman-integration/blob/v1.2/build/libs/postman-integration-1.2.jar

**Comment by cwaazywabbit** [comment]
I am not sure if I still have the project around as it was for a client. I will try to see if it is still around.

**Comment by tagomaru** [comment]
This was resolved at v1.2.
If you have this problem still, please reopen.
